src/quadratic.jl

```diff
@@ -36,9 +36,9 @@ function coefficients(q::Quadratic, N, d)
     symm, sym, symp =  symbol(string("cm_",d)), symbol(string("c_",d)), symbol(string("cp_",d))
     symfx = symbol(string("fx_",d))
     quote
-        $symm = convert(TIndex, .5 * ($symfx - .5)^2)
-        $sym  = convert(TIndex, .75 - $symfx^2)
-        $symp = convert(TIndex, .5 * ($symfx + .5)^2)
+        $symm = convert(TIndex, 1//2 * ($symfx - 1//2)^2)
```

Comment:
Probably not. And besides, we're converting to `TIndex` anyway (which currently defaults to `Float64` for everything except `Rational` and `Float32`) so I guess I'm just paranoid.

This might be something to take up in the manual though, if/when there is a "best practices" section. I'll take a look and see what I can find...